Miniconda3/4.10.3
python: can't open file '02-fetch_gbif_moth_data_serial.py': [Errno 2] No such file or directory
